patches/server/0001-Setup-Gradle-project.patch

```diff
@@ -71,10 +71,11 @@ index 0000000000000000000000000000000000000000..596b4f52bc57eca8cc7a0138a1e36d71
 +    manifest {
 +        val git = Git(rootProject.layout.projectDirectory.path)
 +        val gitHash = git("rev-parse", "HEAD").getText().substring(0, 7)
++        val implementationVersion = System.getenv("BUILD_NUMBER") ?: "\"$gitHash\""
```

Comment:
Is this the right environment variable for TeamCity?

Reply:
Yes, that's why I linked the TeamCity documentation. Every CI system I've used did use it, but doesn't hurt to make sure :)
Not sure what the build number format is configured on your side, but I imagine its this.
![image](https://user-images.githubusercontent.com/46570876/122670517-c9892100-d1c2-11eb-92fa-9b883e52d2b0.png)